internal/checker/check.go

```diff
@@ -74,11 +74,10 @@ func checkAndProcessPanics(
 
 	errChan := importLoopData.ErrChan
 	doesRulePassChan := make(chan bool)
-
 	go func() {
 		defer func() {
 			r := recover()
-			if errChan != nil {
+			if errChan != nil && r != nil {
```

Comment:
I have one more question. Why did you changed this if statement?

Reply:
Because I've come across a situation where `recover()` was returning nil (since no panic had been thrown earlier but an error variable was returned) and I was only getting a message `panic: nil` when this error happened. When I've changed this if statement I started getting a reasonable error message.